### ISSUE TITLE
new column added; implications?

### ISSUE THREAD

**Issue by wyesoslo** [pcm-dpc/COVID-19#462]
**Type of issue**:  richiesta di informazione | richiesta di dati | suggerimento di enhancement (documentazione, label, _etc_)

**Comment by wyesoslo** [comment]
Today you added a new column, variazione_totale_positivi, with data that had formerly been in the column labeled nuovi_attualmente_positivi.  Can you explain what happened?  If on 24 Feb 2020 there had been 221 current cases, and on 25 Feb 2020 there were 311 current cases, how is it that there were 93 new cases instead of 90?  The newly labeled column has what were formerly called new current cases.  The column now called nuovi_attualmente_positivi has numbers that are much higher than the variazione_totale_positivi,column.  Can you make a comment explaining the change?

**Comment by umbros** [comment]
Hi @wyesolo, in this modification we have introduced the data new positives (new_positives) which highlights the real number of people found positive to CoronaVirus. We have perfected the naming of the new positive ones which actually represents the difference between today's positive totals minus yesterday's. The new positives, however, are derived from the calculation of yesterday's and today's total cases.
